sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts

```diff
@@ -312,6 +350,26 @@ describe(
         CSGP: 499,
         XYZ: 444,
       });
+      // Notification record.
+      const notification = await db.notification.findOne({
+        select: {
+          messagePayload: true,
+          createdAt: true,
+        },
+        where: {
+          createdAt: MoreThan(currentTestTime),
+        },
+      });
+      const emailContact = notification.messagePayload["email_address"];
+      const file =
+        notification.messagePayload["personalisation"]["application_file"][
+          "file"
+        ];
+      const fileContent = Buffer.from(file, "base64").toString("ascii");
+      expect(emailContact).toBe(TEST_EMAIL);
+      expect(fileContent).toContain(
```

Comment:
The file content values must be checked to ensure the correct values were generated as per the technical AC:
"Check if the notification attachment has the expected numbers (the generated file can be checked as a whole string)."